src/main/java/dataLoader/jsonDataLoader.java

```diff
@@ -0,0 +1,40 @@
+package dataLoader;
+
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Simple Class that parses data set from a json file
+ * without knowing the files format
+ * & prints out the key value pairs.
+ */
+
+
+
+public class jsonDataLoader {
+
+    private String location = "where the file is located";
+    private File json = new File(location);
```

Comment:
same as above this does not seem to be used anywhere

Reply:
I think I cleared the mess with the variables. Not sure check it out.